tests/components/nobo_hub/test_config_flow.py

```diff
@@ -0,0 +1,275 @@
+"""Test the Nobø Ecohub config flow."""
+from unittest.mock import PropertyMock, patch
+
+from homeassistant import config_entries, setup
+from homeassistant.components.nobo_hub.const import CONF_OVERRIDE_TYPE, DOMAIN
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_configure_with_discover(hass: HomeAssistant) -> None:
+    """Test configure with discover."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
```

Comment:
We don't need to set up the persistent_notification integration anymore.

Reply:
Fixed (in all tests).